### DISCUSSION TITLE
WearOS support

### DISCUSSION THREAD

**Discussion by fortxg** [CatimaLoyalty/Android#364] [Ideas]
Hello there!
Just as the title says, do you think would be possible to add WearOS compatibility?
Thanks!

**Comment by TheLastProject** [comment]
There is already a feature request open for that here: https://github.com/TheLastProject/Catima/issues/25

The thing is, I don't have any WearOS devices, so I can't do this myself. There also seems to be no Open Source library for WearOS support, so this would require Catima to be built in 2 different flavours, one with and one without WearOS support.

**Reply by fortxg** [reply]
Oh, sorry, didn't see the feature request.
Thank you anyway!
